lib/src/navigation.dart

```diff
@@ -110,6 +117,8 @@ class BottomNavScaffold extends ConsumerWidget {
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final currentTab = ref.watch(currentBottomTabProvider);
+    final connectivity = ref.watch(connectivityChangesProvider);
+    final isOnline = connectivity.value?.isOnline ?? false;
```

Comment:
You should not assume the value is false when in loading state, I'd prefer to have `true` by default.

Also the connectivity is only needed for the watch tab, so instead of having the `ref.watch` here, it would be better to wrap the `NavigationDestination` with a `Consumer` widget so only that one rebuilds.

Reply:
It's impossible  cause i need this value in `onDestinationSelected` cb, all what can i do, it wrap `NavigationBar` with Consumer, but I coudn’t do it with `CupertinoTabBar`.